tests/test-titlebar.lua

```diff
@@ -183,6 +197,48 @@ local steps = {
 
         assert(found_font == "sans 10")
 
+        -- Test the events.
+        for _, event in ipairs { "button::press", "button::release", "mouse::move" } do
+            c:connect_signal(event, function()
+                table.insert(events, event)
+            end)
+        end
+
+        next_pos = { x = c:geometry().x + 5, y = c:geometry().y + 5 }
+
+        return true
+    end,
+    click,
```

Comment:
I can't understand what behavior from coverage you want to trigger with these clicks. Can you elaborate?

Reply:
This one is one of the flacky coverage `if`:
![image](https://user-images.githubusercontent.com/340384/139513792-284ff722-b440-4dee-89ff-3c6f48b7074d.png)